crud.py

```diff
@@ -0,0 +1,99 @@
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorCreate, BookCreate
+
+
+def create_author(db: Session, author: AuthorCreate) -> models.Author:
+    db_author = models.Author(**author.model_dump())
```

Comment:
The method `model_dump()` is used here, but it might not exist in the `AuthorCreate` schema. Ensure that `AuthorCreate` has this method or replace it with the appropriate method to convert the schema to a dictionary, such as `dict()`.